firmware/src/rdm.h

```diff
@@ -265,6 +265,14 @@ typedef enum {
 
 
 /**
+ * @brief Open Lighting Project manufacturer PIDs.
+ */
+typedef enum {
+  PID_DEVICE_MODEL = 0x8002,
+  PID_DEVICE_MODEL_LIST = 0x8003
+} OpenLightingManufacturerPID;
```

Comment:
I don't think it worth it since this is just a testing device. We can add them later but I'm trying to minimize the number of PIDs that are intercepted by the rdm_handler.